### PR TITLE
SA342 migration to Module

### PR DESCRIPTION
**Pull request by jdahlblom**

### CODE REVIEW

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+SA342:defineFloat("AUX_FUEL_TANK", 152, { 0, 1 }, "Raw Gauge Values", "Auxiliary Fuel Tank")
+SA342:defineFloat("AM_RADIO_FREQ_100S", 133, { 0, 1 }, "Raw Gauge Values", "AM Radio Frequency 100s")
+SA342:defineFloat("AM_RADIO_FREQ_10S", 134, { 0, 1 }, "Raw Gauge Values", "AM Radio Frequency 10s")
+SA342:defineFloat("AM_RADIO_FREQ_1S", 136, { 0, 1 }, "Raw Gauge Values", "AM Radio Frequency 1s")
+SA342:defineFloat("AM_RADIO_FREQ_TENTHS", 138, { 0, 1 }, "Raw Gauge Values", "AM Radio Frequency Tenths")
+SA342:defineFloat("AM_RADIO_FREQ_HUNDREDTHS", 139, { 0, 1 }, "Raw Gauge Values", "AM Radio Frequency Hundredths")
+
+--Actual Gauge Values
+local function getHDG()
+	local returnValue = (GetDevice(0):get_argument_value(26)) * 360
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua:366]: There are a lot of GetDevice(0) calls that can be replaced by adding dev0 as a function parameter.

**Review by charliefoxtwo — APPROVED** [review]
Personally I'd replace the GetDevice(0) calls, but otherwise this seems fine.

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua (hunk 1 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua:366]: and have a ```local dev0 = GetDevice(0)```?

Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua:366]: Ah I get it, the ```defineIntegerFromGetter``` will supply the dev0.

Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/SA342.lua]: Yup looks good to me!

**Pull request merged**